tests/phpunit/tests/test-object-cache.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+use WP_Syntex\Object_Cache_Annihilator;
+
+class Test_Object_Cache extends PLL_UnitTestCase {
+	private $cache_backup;
+
+	public function set_up() {
+		global $wp_object_cache;
+
+		parent::set_up();
+
+		$this->cache_backup = $wp_object_cache;
+
+		self::factory()->language->create_many( 2 );
+
+		// Drop in the annihilator.
+		copy( PLL_TEST_DATA_DIR . 'object-cache-annihilator.php', WP_CONTENT_DIR . '/object-cache.php' );
+		wp_using_ext_object_cache( true );
+		$wp_object_cache = new Object_Cache_Annihilator();
+
+		$options       = $this->create_options( array( 'default_lang' => 'en' ) );
+		$model         = new PLL_Model( $options );
+		$links         = $model->get_links_model();
+		$this->pll_env = new PLL_Frontend( $links, $model );
+		$this->pll_env->init();
+		$GLOBALS['polylang'] = $this->pll_env;
+	}
+
+	public function tear_down() {
+		global $wp_object_cache;
+
+		// Annihilate the annihilator.
+		$wp_object_cache->flush();
+		unlink( WP_CONTENT_DIR . '/object-cache.php' );
+		$wp_object_cache = $this->cache_backup;
+
+		parent::tear_down();
+	}
+
+	public function test_object_cache_unavailable() {
```

Comment:
The test doesn't fail if I comment the 2 lines added to `PLL_Model` :s

Reply:
Fixed in dda9d27b371a94a532f7c9d7e54b707f231a116c.
It should fail if the fix is disabled now. I don't know where I fucked things up during my development...